user/caribou/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "caribou"
+pkgver = "0.4.21"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-gtk2-module"]
+hostmakedepends = [
+    "automake",
+    "gettext-devel",
+    "gobject-introspection",
+    "intltool",
+    "pkgconf",
+    "slibtool",
+    "xsltproc",
+]
+makedepends = [
+    "at-spi2-core-devel",
+    "clutter-devel",
+    "glib-devel",
+    "gtk+3-devel",
+    "libgee-devel",
+    "libx11-devel",
+    "libxklavier-devel",
+    "libxml2-devel",
+    "libxtst-devel",
+    "python-gobject-devel",
+]
+depends = ["clutter"]
+pkgdesc = "Text entry and UI navigation application"
+maintainer = "Earldridge Jazzed Pineda <earldridgejazzedpineda@gmail.com>"
+license = "LGPL-2.1-or-later"
+url = "https://wiki.gnome.org/Projects/Caribou"
+source = f"https://download.gnome.org/sources/caribou/{pkgver[:-3]}/caribou-{pkgver}.tar.xz"
```

Comment:
```suggestion
source = f"https://download.gnome.org/sources/caribou/{".".join(pkgver.split(".")[0:2)}/caribou-{pkgver}.tar.xz"
```

deleting the last three chars would just break when it becomes 0.5.3 or similar

Reply:
Caribou hasn't been updated in a while, but I will add this change in there just in case it gets updated.